staging/src/k8s.io/cli-runtime/pkg/genericclioptions/print_flags.go

```diff
@@ -136,7 +136,7 @@ func (f *PrintFlags) AddFlags(cmd *cobra.Command) {
 	f.TemplatePrinterFlags.AddFlags(cmd)
 
 	if f.OutputFormat != nil {
-		cmd.Flags().StringVarP(f.OutputFormat, "output", "o", *f.OutputFormat, fmt.Sprintf("Output format. One of: %s.", strings.Join(f.AllowedFormats(), "|")))
+		cmd.Flags().StringVarP(f.OutputFormat, "output", "o", *f.OutputFormat, fmt.Sprintf(`Output format. One of: "%s".`, strings.Join(f.AllowedFormats(), " ")))
```

Comment:
Let's change this to: explanation + (examples), like so: 
```
 Output format. (json, yaml, name, go-template, ...)
```
wdyt?

Reply:
It looks great with that, as long as there is space, the wrapper will be able to split the line